src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -15,9 +16,11 @@ public class MovieSession {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
-    @ManyToOne(fetch = FetchType.LAZY)
+    @OneToOne(fetch = FetchType.LAZY)
```

Comment:
```suggestion
    @ManyToOne(fetch = FetchType.LAZY)
```
Can be more sessions for one movie